CHANGELOG.rst

```diff
@@ -1,10 +1,19 @@
 Changelog
 =========
 
-0.6.0 (unreleased)
+1.0.0 (unreleased)
 ------------------
 
-- Nothing changed yet.
+**Breaking changes**
+
+- By default record attachments now contains a relative location to
```

Comment:
We probably need to have a meeting @leplatrem you and I about this.

The base line is that we would like to have a unique API to handle this gracefully without having to check for one behavior or the other on client side.

The thing is since you will probably serve static file from a CDN, the base url can change.
Having the full URL in the record can be a problem because if you change it you need to update all records.

It can also save space especially for mobile apps to have relative links.

Having the full url is like de-normalizing the base-url in each record.

Our Kinto client are already checking the capability in the root urls so the call on the root endpoint is already present and can be reuse to get the base-url.

@leplatrem was in favor on removing the setting all-together so that we do not have do have a protocol conditional behavior on client side.

I proposed to keep it so that previous app can continue to use this behavior. Be for the reason I mentioned we think it is better to have it as the default behavior and for the reason you presented I think it is good if we can let people keep absolute url for certain use case.